types/amap-js-api-tools/tslint.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-bad-reference": false
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
do you need this rule? Why it was added?

---

Not really,  I have seen this error when run npm test
```shell
node --require source-map-support/register node_modules/@definitelytyped/dtslint/ types "amap-js-api-tools"

dtslint@0.0.133

Could not find implementations for the following rules specified in the configuration:
    no-bad-reference
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```

---

This looks like a bug in dtslint; this rule was converted to ESLint but there's still a reference to it in a config.